File: MyApplication/app/src/main/java/com/example/myapplication2/WSServicios.java
package com.example.myapplication2;
import com.google.gson.annotations.Expose;
import com.google.gson.annotations.SerializedName;



public class WSServicios {

    @SerializedName("id")
    @Expose
    private Integer id;
    @SerializedName("cantidad")
    @Expose
    private Integer cantidad;
    @SerializedName("precio")
    @Expose
    private Double precio;
    @SerializedName("descripcion")
    @Expose
    private String descripcion;

    public Integer getId() {
        return id;
    }

    public void setId(Integer id) {
        this.id = id;
    }

    public Integer getCantidad() {
        return cantidad;
    }

    public void setCantidad(Integer cantidad) {
        this.cantidad = cantidad;
    }

    public Double getPrecio() {
        return precio;
    }

    public void setPrecio(Double precio) {
        this.precio = precio;
    }

    public String getDescripcion() {
        return descripcion;
    }

    public void setDescripcion(String descripcion) {
        this.descripcion = descripcion;
    }
}

File: kotlin_proyect/app/src/main/java/com/example/kotlin_proyect/WSServicios.kt
package com.example.kotlin_proyect

import com.google.gson.annotations.Expose
import com.google.gson.annotations.SerializedName

class WSServicios {
    @SerializedName("id")
    @Expose
    private var id: Int? = null

    @SerializedName("cantidad")
    @Expose
    private var cantidad: Int? = null

    @SerializedName("precio")
    @Expose
    private var precio: Double? = null

    @SerializedName("descripcion")
    @Expose
    private var descripcion: String? = null

}